config/settings.yml

```diff
@@ -45,3 +45,7 @@
         :ems_refresh_worker_amazon_network: {}
         :ems_refresh_worker_amazon_ebs_storage: {}
         :ems_refresh_worker_amazon_s3: {}
+:blacklisted_event_names:
```

Comment:
@durandom  - is the merge of ManageIQ/manageiq#13942 imminent?
Why would we not want this editable?